CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 
 ### Features
 
+- `[jest-worker, jest-haste-map, jest-runner, jest-reporters]` Fix: jest-worker: should not expose `.default` babel interop ([#10623] (https://github.com/facebook/jest/pull/10623))
```

Comment:
sorry, _this_ one should be under master 😅 

If you tick this box in the lower right I can fix all of these little nits myself

![image](https://user-images.githubusercontent.com/1404810/95733254-24589900-0c82-11eb-956e-42534e983740.png)

Reply:
@SimenB i cant see this option

Reply:
hmm, might be it needs to be enabled by repoo admins. @mkcode might know? Giving me push permission to the MLH for should also fix it so I'm able to push fixups to your PRs